open_feature/flag_evaluation/flag_evaluation_options.py

```diff
@@ -0,0 +1,10 @@
+import typing
+from dataclasses import dataclass, field
+
+from open_feature.hooks.hook import Hook
+
+
+@dataclass
+class FlagEvaluationOptions:
```

Comment:
I can't see this in the [list of items for spec 0.2.0](https://github.com/open-feature/python-sdk/issues/7)? Is this something from a later update to the spec - or it was just missing from the GH issue?

Reply:
Flag evaluation options are needed for the hook providers being handled in the correct order

Reply:
Understood.